### PR TITLE
Feat/posh page

### PR DESCRIPTION
**Pull request by carmeltsadok**

### CODE REVIEW

File: mockserver/nodeGroup/nodeGroup.manager.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ import Role from '../user/enum.js';
 class NodeGroupManager {
   static async getNodeGroups(req, res) {
     const requester = jwt.decode(req.headers.authorization.split(' ')[1]);
-    const unitNodesGroup = nodeGroups.filter((nodeGroup) => nodeGroup.unitId === requester.unitId);
+    const { unitId } = req.query;
+    const unitNodesGroup = unitId
+    ? nodeGroups.filter((nodeGroup) => nodeGroup.unitId === unitId)
+    : nodeGroups.filter((nodeGroup) => nodeGroup.unitId === requester.unitId);
```
Comment (PCOffline) [inline mockserver/nodeGroup/nodeGroup.manager.js:12]: Remove redundant code:

```suggestion
    const unitNodesGroup = nodeGroups.filter((nodeGroup) => nodeGroup.unitId === (unitId ?? requester.unitId) );
```

File: src/common/ScheduleHeader/ScheduleHeader.styles.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const useStyles = makeStyles((theme) => ({
   item: {
     margin: '0 1em',
     [theme.breakpoints.down('sm')]: {
-      margin: '0.5em 0',
+      margin: '0.2em 0 !important',
```
Comment (PCOffline) [inline src/common/ScheduleHeader/ScheduleHeader.styles.js:26]: Try creating a more specific rule instead of using `!important`

**Review by PCOffline — CHANGES_REQUESTED** [review]

File: src/common/ScheduleHeader/ScheduleHeader.styles.js (hunk 2 above)
Comment (carmeltsadok) [reply src/common/ScheduleHeader/ScheduleHeader.styles.js:26]: it happened to me before.. I have to put the impotant for ut to work in all cases

File: mockserver/nodeGroup/nodeGroup.manager.js (hunk 1 above)
Comment (carmeltsadok) [reply mockserver/nodeGroup/nodeGroup.manager.js:12]: it is mock. who cares?

**Review by PCOffline — APPROVED** [review]

**Review by maykremer — APPROVED** [review]

**Review by Delitefully — APPROVED** [review]

**Review by PCOffline — APPROVED** [review]

**Pull request merged**